src/app/pages/list/status-list/status-list.component.html

```diff
@@ -5,22 +5,182 @@
     <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Status List</div>
     <pxb-spacer></pxb-spacer>
 </mat-toolbar>
-<mat-list class="alert" *ngIf="data.length > 0">
-    <pxb-info-list-item
-        *ngFor="let item of data; let index = index"
-        [statusColor]="setAlarm(item.status) ? colors.red[500] : ''"
-        [avatar]="true"
-    >
-        <mat-icon
-            pxb-icon
-            [style.color]="getIconColor(item.status)"
-            [style.backgroundColor]="item.status === 'alarm' ? colors.red[500] : ''"
-            >{{ item.icon }}</mat-icon
-        >
-        <span pxb-title>{{ item.name }}</span>
-        <span pxb-subtitle>{{ item.details }}</span>
-        <span pxb-right-content *ngIf="setAlarm(item.status)">
-            <pxb-list-item-tag [backgroundColor]="colors.red[500]" label="new"></pxb-list-item-tag>
-        </span>
-    </pxb-info-list-item>
-</mat-list>
+<section class="list-section">
+    <div class="list-example">
+        <mat-accordion class="accordion-panel">
+            <mat-expansion-panel [expanded]="true" [class.expansion-panel-mobile]="isSmall">
+                <mat-expansion-panel-header>
+                    <mat-panel-title>
+                        With Time Stamps, with Title+Subtitle+Info
+                    </mat-panel-title>
+                </mat-expansion-panel-header>
+                <mat-list class="list-section">
+                    <pxb-info-list-item
+                        divider="partial"
+                        [statusColor]="colors.red[500]"
+                        chevron="true"
```

Comment:
Since `chevron` is also a boolean, it requires the [brackets] around it.

Otherwise `chevron="false"` would still render the chevrons since it interprets the string "false" as truthy.